test/snapshots/main.js.md

```diff
@@ -33,6 +33,7 @@ Generated by [AVA](https://avajs.dev).
         },
         env: {},
         id: 'sapper',
+        logos: undefined,
```

Comment:
Question: Are we ok with leaving `logos` as `undefined` until they're all added?